### PR TITLE
chore(deps): remove peerDependencies from tests-playwright package

### LINKED ISSUE #8839: Update @podman-desktop/tests-playwright peerDependencies or remove them from the package
### Is your enhancement related to a problem? Please describe

Running the tests from external extension is broken due to usage of different version in the package peerDependencies and devDependencies in `@podman-desktop/tests-playwright`.

### Describe the solution you'd like

Update and/or remove the dependencies that are not needed.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by odockal**
### What does this PR do?
Removes the peerDependencies from @podman-desktop/tests-playwright package. 
### What issues does this PR fix or reference?
#8839 

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Review by benoitf — COMMENTED** [review]
not sure why you're using peerDependencies without a range ? also maybe they're useless

File: tests/playwright/package.json [hunk 1]
```diff
@@ -20,14 +20,13 @@
...
-    "electron": "29.1.4"
+    "@playwright/test": "1.47.0",
+    "electron": "32.0.2"
   },
   "devDependencies": {
     "@playwright/test": "1.47.0",
     "@types/node": "^20",
     "electron": "32.0.2",
-    "typescript": "^5.5.4",
-    "vitest": "^2.0.5"
```
Comment (axel7083) [inline tests/playwright/package.json]: Removing vitest on purpose ?

**Comment by odockal** [comment]
> not sure why you're using peerDependencies without a range ? also maybe they're useless

I will try to remove them, I am not sure if it can break something outside podman-desktop.

File: tests/playwright/package.json (hunk 1 above)
Comment (odockal) [reply tests/playwright/package.json]: Yes. Vitest is not used in e2e tests anymore. Though it is still used for running all other tests in root package.

Comment (odockal) [reply tests/playwright/package.json]: but this change might be in single PR, since there is more to remove. I will revert for this PR.

**Review by benoitf — APPROVED** [review]

**Pull request merged**